agasc/tests/test_agasc.py

```diff
@@ -37,9 +37,15 @@
     assert os.path.exists(ascrc_file)
     ascds_env = Ska.Shell.getenv('source {} -r release'.format(ascrc_file), shell='tcsh')
     assert 'ASCDS_BIN' in ascds_env
+    # Make a mp_get_agasc cmd line that just returns that command's status
+    cmd = 'mp_get_agasc -r 10 -d 20 -w 0.01 >& /dev/null; echo $?'
+    lines = Ska.Shell.tcsh(cmd, env=ascds_env)
+    # Check command status is 0
+    assert lines[0] == "0"
```

Comment:
I don't immediately see what we gain by making this work with Spawn.  It also looks like the "native" status check in run_shell wants the cmd to return a prompt, and by the time I construct that shell cmd to do that, it seems like explicitly returning the status as done here is basically equivalent.